parquet-column/src/main/java/org/apache/parquet/io/api/Binary.java

```diff
@@ -262,12 +313,17 @@ public void writeTo(OutputStream out) throws IOException {
     }
 
     @Override
+    public byte[] getBytesUnsafe() {
+      return getBytes();
```

Comment:
If the buffer isn't reused, then it is safe to cache a copy of the bytes and use it each time unsafe is called rather than copying all the time.